Graph-a-Single-SNMP-OID.md

```diff
@@ -34,5 +34,9 @@ When finished filling in values for these fields, click the Create button. Your
 new graph can now be accessed through the Graph Management page or the Graphs
 tab inside of Cacti.
 
+#### Note on graphing single OID's with spine
+
+Spine will only proccess a numeric OID i.e 1.3.6.1.1.1 spine will not parse OID's such as enterprise.1.3.6.1.1.1 
```

Comment:
The repeatative nature of the sentence is still there.  It should probably read something like, 

> In order to ensure that Spine will process single OID's correct, they should be written using numeric format separated by dots, such as `1.3.6.1.4.1.9.9.97`.  At this time, ASN's (named OID's) such as `enterprise.9.9.97` (Cisco Switch) or `enterprises.cisco.ciscoMgnt.ciscoSwitchEngineMIB` are not supported.

Reply:
are you ok If I use what you wrote? seems pretty through

Reply:
Yeah why not.

Reply:
cool thanks
Added